.gitlab-ci.yml

```diff
@@ -192,6 +192,18 @@ publish:release-rx:
     - git fetch --depth=1 origin master
     - ./gradlew :dd-sdk-android-rx:bintrayUpload --stacktrace --no-daemon
 
+publish:release-sqlite:
```

Comment:
Since the SQLiteOpenHelper is part of the core Android Framework, I'm not sure we actually need another module just for that. 
My take is that we should create a distinct artifact to avoid adding unneeded dependencies in the core library.